libr/bin/p/bin_xnu_kernelcache.c

```diff
@@ -723,6 +723,11 @@ static RList *entries(RBinFile *bf) {
 	}
 
 	RKernelCacheObj *kobj = (RKernelCacheObj*) obj->bin_obj;
+	ut64 entry_vaddr = kobj->mach0->entry;
```

Comment:
kobj can be null?

Reply:
well, there's already a check right before